apps/ledger-live-mobile/src/screens/SendFunds/02-SelectRecipient.tsx

```diff
@@ -225,12 +226,22 @@ export default function SendSelectRecipient({ navigation, route }: Props) {
               />
             </View>
             {(error || warning) && (
-              <LText
-                style={[styles.warningBox]}
-                color={error ? "alert" : warning ? "orange" : "darkBlue"}
-              >
-                <TranslatedError error={error || warning} />
-              </LText>
+              <>
+                <LText
+                  style={[styles.warningBox]}
+                  color={error ? "alert" : warning ? "orange" : "darkBlue"}
+                >
+                  <TranslatedError error={error || warning} />
+                </LText>
+                <View
+                  style={{
+                    display: "flex",
+                    alignItems: "flex-start",
+                  }}
+                >
+                  <SupportLinkError error={error} type="alert" />
+                </View>
```

Comment:
No way (or no need?) to include this conditionally? Won't it change standard display on other components?

Reply:
SupportLinkError have a conditionnal inside him so it should return null, it's an hidden conditionnal but I can change it if needed